src/ralph/domains/models/domains.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from dj.choices import Choices
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.assets.models import BaseObject
+from ralph.assets.models.assets import BusinessSegment
+from ralph.back_office.models import AssetHolder
+from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+
+class DomainRegistrant(
+    PermByFieldMixin,
+    NamedMixin,
+    TimeStampMixin,
+    models.Model,
+):
+    pass
+
+
+class DomainStatus(Choices):
+    _ = Choices.Choice
+
+    active = _('Active')
+    pending_lapse = _('Pending lapse')
+    pending_transfer = _('Pending transfer away')
+    lapsed = _('Lapsed (inactive)')
```

Comment:
NO. "lapsed" is used in domain management dictionary. Expired is the process "in the middle"